pkg/commands/docker.go

```diff
@@ -125,3 +123,33 @@ func (t *dockerBuildTask) Push(runner build.Runner) error {
 	}
 	return runner.ExecuteCmd("docker", []string{"push", t.pushTo})
 }
+
+// PopulateDigests populates digests on dependencies
+func PopulateDigests(runner build.Runner, dependencies []build.ImageDependencies) error {
+	for _, entry := range dependencies {
+		if err := queryDigest(runner, entry.Image); err != nil {
+			return err
+		}
+		if err := queryDigest(runner, entry.Runtime); err != nil {
+			return err
+		}
+		for _, buildtime := range entry.Buildtime {
+			if err := queryDigest(runner, buildtime); err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+func queryDigest(runner build.Runner, reference *build.ImageReference) error {
+	refString := reference.String()
+	line, err := runner.QueryCmd("docker", []string{
+		"image", "ls", "--digests", "--format", "\"{{ .Digest }}\"", refString,
```

Comment:
{{ .Digest }} what is the extra space for?

Reply:
I always leave the space when I do the query but I suppose I do not need it.